README.md

```diff
@@ -1,286 +1,116 @@
 # React components for TalkJS
 
-This library makes it easy to use the [TalkJS](https://talkjs.com) pre-built chat UIs inside a React web application.
+This library makes it easy to use [TalkJS](https://talkjs.com) pre-built chat UIs inside a React web application.
```

Comment:
Two small suggestions:
- Maybe say "The `@talkjs/react` library" here, to make it super explicit in the opening sentence?
- Maybe could remove "pre-built chat UIs", so that it becomes just "...makes it easy to use [TalkJS](https://talkjs.com) inside..."? As it makes it easy to use the entire TalkJS offering, not only the pre-built chat UI bit?

Reply:
I've gone with https://github.com/talkjs/talkjs-react/pull/5/commits/072c5e919e49abde24ca6bb1ac68b3f306403479, wanted to still emphasise that the library is mainly for the chat UI components. Hopefully clearer now!